README.md

```diff
@@ -12,3 +12,24 @@ Discovery service doesn't store any data, all data is ephemeral and is kept only
 Node information is expired (if not updated) after 30 minutes.
 Discovery service doesn't see actual node information, it only stores and updates encrypted blobs.
 Discovery data should be submitted encrypted by the client, and service doesn't have the encryption key.
+
+The repo has been split into 3 parts(as they have differnt source code licenses), namely:
+- The current repo
+- [Discovery client](https://github.com/talos-systems/discovery-client), contains the client code to interact with the server
+- [Discovery api](https://github.com/talos-systems/discovery-api/), provides various endpoints to access the server
```

Comment:
I would probably suggest to rephrase this line to say that `discovery-api` contains protobuf definitions for the discovery service

Reply:
oh, @smira was faster :D You can pick his suggestions for both